app/pages/cif/project-revision/[projectRevision]/view.tsx

```diff
@@ -15,12 +15,27 @@ import readOnlyTheme from "lib/theme/ReadOnlyTheme";
 import { getLocaleFormattedDate } from "lib/theme/getLocaleFormattedDate";
 import useShowGrowthbookFeature from "lib/growthbookWrapper";
 import NotifyModal from "components/ProjectRevision/NotifyModal";
+import RevisionStatusWidget from "components/ProjectRevision/RevisionStatusWidget";
 
-const createProjectRevisionViewSchema = (allRevisionTypes) => {
-  const schema = projectRevisionSchema;
+const createProjectRevisionViewSchema = (
+  allRevisionTypesEdges: {
+    readonly edges: readonly {
+      readonly node: {
+        readonly type: string;
+      };
+    }[];
+  },
+  allRevisionStatusesEdges: readonly {
+    readonly node: {
+      readonly name: string;
+      readonly isAmendmentSpecific: boolean;
+    };
+  }[]
```

Comment:
- the `readonly` is unnecessary here
- we can remove the edges key to make both arguments consistent in type

```suggestion
const createProjectRevisionViewSchema = (
  allRevisionTypesEdges: {
     node: {
       type: string;
      };
  }[],
  allRevisionStatusesEdges: {
   node: {
     name: string;
     isAmendmentSpecific: boolean;
    };
  }[]
```